modules/admin_manual/pages/configuration/server/occ_commands/app_commands/richdocuments.adoc

```diff
@@ -0,0 +1,81 @@
+= Collabora Online / Secure View
+
+Marketplace URL: {oc-marketplace-url}/apps/richdocuments[Collabora Online]
+
+Collabora Online allows you to work with all kinds of collabora office documents inside ownCloud for improved productivity. For details please see the
+xref:enterprise/collaboration/collabora_secure_view.adoc[Collabora Online / Secure View] documentation.
+
+`config:app` commands to configure the Collabora Online app.
+
+[source,console]
+----
+ config:app:get|set
+ richdocuments wopi_url            WOPI Server URL
+ richdocuments secure_view_option  Enable Secure View
+ richdocuments watermark_text      Watermark pattern displayed in the document
+ richdocuments secure_view_open_action_default  Open documents in Secure View with watermark by default
+----
+
+== App Configuration
+
+All app configurations are set and queried with the `config:app` command set. The examples below set a value. To query a value use `config:app:get` and the corresponding key without any options or attributes.
+
+== Set the WOPI URL
+
+WOPI Server URL
+
+The following example command sets the WOPI Server to [your-host-public-ip]:8098
```

Comment:
Q: because the value set from the _name_ is a url, does it need to be an IP address or can it also be a domain name.
- If it _must_ be IP, this should be noted as restriction
- If it _must_ be IP, is http/https necessary/needed?
- If it can be both, it must be documented
- If domain name can be used, are there restriction in the use of http/https

Reply:
I think admin knows best what is his URL , but URL should be https://datatracker.ietf.org/doc/html/rfc3986

Reply:
ok, I will improve the text to make this more clear.
Asking again, there is NO restriction in using http or https by oC

Reply:
yes it is an example. there is no http/https restriction.